types/react-virtualized/dist/es/List.d.ts

```diff
@@ -73,60 +90,44 @@ export type ListProps = GridCoreProps & {
  * This component renders a virtualized list of elements with either fixed or dynamic heights.
  */
 export class List extends PureComponent<ListProps> {
-    static propTypes: {
-        'aria-label': Requireable<string>,
-        autoHeight: Requireable<boolean>,
-        className: Requireable<string>,
-        estimatedRowSize: Validator<number>,
-        height: Validator<number>,
-        noRowsRenderer: Validator<() => JSX.Element>,
-        onRowsRendered: Validator<(params: IndexRange) => void>,
-        overscanRowCount: Validator<number>,
-        onScroll: Validator<(params: { clientHeight: number, scrollHeight: number, scrollTop: number }) => void>,
-        overscanIndicesGetter: Validator<OverscanIndicesGetter>,
-        rowHeight: Validator<number | ((params: Index) => number)>,
-        rowRenderer: Validator<ListRowRenderer>,
-        rowCount: Validator<number>,
-        scrollToAlignment: Validator<Alignment>,
-        scrollToIndex: Validator<number>,
-        scrollTop: Requireable<number>,
-        style: Validator<React.CSSProperties>,
-        tabIndex: Requireable<number>,
-        width: Validator<number>
-    };
-
     static defaultProps: {
-        estimatedRowSize: 30,
-        noRowsRenderer: () => null,
-        onRowsRendered: () => null,
-        onScroll: () => null,
-        overscanRowCount: 10,
-        scrollToAlignment: 'auto',
-        scrollToIndex: -1,
-        style: {}
+        autoHeight: false;
+        estimatedRowSize: 30;
+        onScroll: () => {};
```

Comment:
Why not `=> void`?

Reply:
thanks for the hint, that was a copy paste mistake